src/authentication/types/AuthenticationResult.ts

```diff
@@ -1,6 +1,8 @@
-export interface AuthenticationResult {
+type AuthenticationResultStructure = {
     token: string;
     login: string;
     firstName: string;
     lastName: string;
 }
+
+export type AuthenticationResult = Readonly<AuthenticationResultStructure>;
```

Comment:
Probably but I first did it like this so you are allowed to used the non readonly type as well. But since we never use it, just throw it away?
It is a nice example of using the Readonly mapped type instead of the 'readonly' on every property...
What do you think?